lib/server.js

```diff
@@ -0,0 +1,112 @@
+const core = require('@actions/core')
+
+async function foundChannel(slackClient, channelName) {
```

Comment:
Autant dire que cela renvoie un id !
```suggestion
async function channelId(slackClient, channelName) {
```
ou
```suggestion
async function getChannelId(slackClient, channelName) {
```
vu que ça sort en erreur s'il n'y en a pas